### DISCUSSION TITLE
Error in "App Color" slider

### DISCUSSION THREAD

**Discussion by bios4** [devemux86/cruiser#320] [Q&A] [answered]
Current version 3.0.32

When trying to freely define the application color (Settings > Application > Appearance > App Color), the values selected are not taken over in certain ranges but set to abritrary values by the app itself.

Examples:
- RGB(43,97,191) --> are set to RGB(45,90,195) after hitting OK
- RGB(61,77,107) --> are set to RGB(60,75,105) after hitting OK
- RGB(203,107,41) --> are set to RGB(210,105,45) after hitting OK

This ONLY happens in the "App Color" slider!
When selecting the very same RGB values in e.g. the symbols sliders the values are taken over as expected.

---

Additional question: is there a chance to get an input field where one can directly insert RGB values?
Operating the sliders and hitting the exact numbers is quite cumbersome at times. ;-)

**Comment by devemux86** [comment]
App color is adjusted to the supported nearest color scheme.
The range is extremely small, so visually it has no difference.

> is there a chance to get an input field where one can directly insert RGB values?

This could be useful, I will see what can be done. :slightly_smiling_face:

**Reply by bios4** [reply]
Thanks for your reply...

Because it is possible to add detailed color settings in the symbols sliders I thought this must be an error in the app color slider.
So does that mean that it is not POSSIBLE at all to have this behaviour available also for the app color slider?

**Reply by devemux86** [reply, marked as answer]
Android does not support custom app (accent) color, like other operating systems.
So I used a trick in Cruiser to implement it and it can only work like this.
